plugins/unix-socket.js

```diff
@@ -0,0 +1,97 @@
+var toPull = require('stream-to-pull-stream')
+var net = require('net')
+var fs = require('fs')
+var path = require('path')
+
+// hax on double transform
+var started = false
+
+module.exports = function (opts) {
+  const config = require('ssb-config/inject')(process.env.ssb_appname)
+  const socket = path.join(config.path, 'socket')
+  const addr = 'unix:' + socket
+  
+  function toDuplex (str) {
+    var stream = toPull.duplex(str)
+    stream.address = addr
+    stream.remote = 'unix'
+    stream.auth = { allow: null, deny: null }
```

Comment:
this shouldn't be here. doesn't noauth provide this?

Reply:
Yep, thanks!